doc/explanations/min-utxo-alonzo.rst

```diff
@@ -0,0 +1,99 @@
+|                     |                 |                 |                   |                  |                  |                                 |
+|                     | one 0-character | one 1-character | three 1-character | one 0-character  | one 1-character  | ninety-six 1-character          |
+|                     |                 |                 |                   |                  |                  |                                 |
+|                     | asset name (i)  | asset name (ii) | asset names (iii) | name (iv)        | name for each (v)| names between them (total) (vi) |
++---------------------+-----------------+-----------------+-------------------+------------------+------------------+---------------------------------+
+| size of value       | 11              | 12              | 15                | 16               | 17               | 173                             |
++---------------------+-----------------+-----------------+-------------------+------------------+------------------+---------------------------------+
+| ``minUTxO``         | 1,310,316       | 1,344,798       | 1,448,244         | 1,482,726        | 1,517,208        | 6,896,400                       |
++---------------------+-----------------+-----------------+-------------------+------------------+------------------+---------------------------------+
+| ``minUTxO`` (in ada)| 1.310316        | 1.344798        | 1.448244          | 1.482726         | 1.517208         | 6.896400                        |
++---------------------+-----------------+-----------------+-------------------+------------------+------------------+---------------------------------+
+
+** WITH datum hash: **
+
++---------------------+-----------------+--------------------+------------------+
+|                     | One policyID,   | One PolicyID,      | Two PolicyIDs,   |
+|                     |                 |                    |                  |
+|                     | one 0-character | three 32-character | one 0-character  |
+|                     |                 |                    |                  |
+|                     | asset name (vii)|  asset name (viii) | name (ix)        |
++---------------------+-----------------+--------------------+------------------+
+| size of value       | 11              |  26                | 16               |
++---------------------+-----------------+--------------------+------------------+
+| ``minUTxO``         | 1,655,136       | 2,172,366          | 1,827,546        |
++---------------------+-----------------+--------------------+------------------+
+| ``minUTxO`` (in ada)| 1.655136        |  2.172366          | 1.827546         |
++---------------------+-----------------+--------------------+------------------+
+
+
+* (i) : ``6 + FLOOR(((1 * 12) + 0 + (1 * 28) + 7)/8, 1) = 11``
```

Comment:
These calculations are not explained at all by the above document. In fact, since there is a formula above, the reader could think that these are examples of the formulas above (which was my first thought!)